reference-implementation/lib/transform-stream.js

```diff
@@ -277,7 +240,11 @@ class TransformStreamDefaultController {
       throw defaultControllerBrandCheckException('error');
     }
 
-    TransformStreamError(this._controlledTransformStream, reason);
+    if (this._controlledTransformStream._readable._state !== 'readable') {
+      throw new TypeError('TransformStream is not in a state that can be errored');
+    }
+
+    TransformStreamDefaultControllerError(this, reason);
```

Comment:
Since TransformStreamDefaultControllerError is so short, can we just inline it here?

Reply:
I was thinking it might be useful as a hook for other standards, but since they are more likely to just throw from transform(), maybe not?

Reply:
Oh, I see, in parallel to the ones for other controllers. Yeah, maybe keep it, in which case LGTM for the whole patch.